www/html/ci/application/views/landing.php

```diff
@@ -334,367 +237,307 @@ function googleTranslateElementInit() {
 			</div>
 		</div>
 	</section>
-
 </div>
 
 <!--SMASH-->
 <div class="container kt-tabs-component-content clean-container-theme strictly-no-padding" id="smash">
 	<div class="jumbotron strictly-no-background-color">
 		<div class="container">
-		  <div class="text-center">
-		  	<h1 class="section-heading ">I'll just throw other stuff here.</h1><!--'-->
-		  </div>
-		  <div class="col-sm-12 maintenance-info-box">
-			<h3>Follow me on <a href="https://twitter.com/kTmochi">Twitter</a></h3>
-			<h3>Listen to some music on spotify that I listen to:</h3>
-			<iframe src="https://embed.spotify.com/?uri=spotify%3Auser%3Aktorii%3Aplaylist%3A4z1Tkn4FCpt5ARe81ev1C6" width="300" height="380" frameborder="0" allowtransparency="true"></iframe>
-			<iframe src="https://embed.spotify.com/?uri=spotify%3Auser%3Aktorii%3Aplaylist%3A5iRIXZAwETSNzgjZihwLGi" width="300" height="380" frameborder="0" allowtransparency="true"></iframe>
-		  </div>
-	  	</div>
+			<div class="text-center">
+				<h1 class="section-heading">I'll just throw other stuff here.</h1><!--'-->
+			</div>
+			<div class="col-sm-12 maintenance-info-box">
+				<h3>Follow me on <a href="https://twitter.com/kTmochi">Twitter</a></h3>
+				<h3>Listen to some music on spotify that I listen to:</h3><iframe frameborder="0" height="380" src="https://embed.spotify.com/?uri=spotify%3Auser%3Aktorii%3Aplaylist%3A4z1Tkn4FCpt5ARe81ev1C6" width="300"></iframe> <iframe frameborder="0" height="380" src="https://embed.spotify.com/?uri=spotify%3Auser%3Aktorii%3Aplaylist%3A5iRIXZAwETSNzgjZihwLGi" width="300"></iframe>
+			</div>
+		</div>
 	</div>
-
 </div>
 
 <!--MATH-->
 <div class="container kt-tabs-component-content clean-container-theme strictly-no-padding" id="math">
 	<div class="container bg-light-gray">
 		<div class="col-lg-12 text-center div-extra-padding">
-			<h1 id="" class="section-heading">Math</h1>
+			<h1 class="section-heading" id="">Math</h1>
```

Comment:
You do not need to switch the position of id and class. We'll keep it for now though.